### PR TITLE
PHPC-489: Fixed some links, and added new links to the new resources

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

File: source/drivers/php.txt [hunk 1]
```diff
@@ -16,69 +16,29 @@ To access MongoDB from PHP you need:
...
-----------------------
+Drivers
+-------
 
-\*NIX
-~~~~~
+The currently maintained driver is the **mongodb** extension available from
+PECL. This driver can be used stand-alone, although it is very bare-bones. You
+should however use the driver through the complimentary *PHP library*, which
+implements a more convenient API on top of the bare-bones driver.
```
Comment (jmikola) [inline source/drivers/php.txt]: "You should consider using the driver with the complimentary *PHP library*, which implements a more full-featured API atop the bare-bones driver."

File: source/drivers/php.txt [hunk 2]
```diff
@@ -16,69 +16,29 @@ To access MongoDB from PHP you need:
...
 
-Run:
+For further information on the architecture, please refer to this
+`blog post <https://www.mongodb.com/blog/post/call-feedback-new-php-and-hhvm-drivers>`_.
 
-.. code-block:: sh
+There is also an older legacy driver called **mongo**. For all future
+projects you should use the **mongodb** extension with the *PHP library*. The
```
Comment (jmikola) [inline source/drivers/php.txt]: Rewrite as "New projects should use the"

**Comment by derickr** [comment]
Ok, good to go from our side!

**Comment by kay-kim** [comment]
merged and published.

**Pull request closed without merging**